docs/source/tutorial.rst

```diff
@@ -0,0 +1,241 @@
+            (BLOCKED, "Blocked"),
+            (INACTIVE, "Inactive"),
+        ]
+
+        staus = models.CharField(
+            max_length=2,
+            choices=STATUS_CHOICES,
+        )
+
+        last_review = models.DateField(null=True, blank=True)
+
+        def is_at_risk(self):
+            return self.status in {self.BLOCKED, self.INACTIVE}
+
+        def __str__(self):
+            return self.question_text
+
+
+
+Then add the project, the ``projects`` module and Neapolitan to the beginning of
+``INSTALLED_APPS`` in ``settings.py``:
+
+ .. code-block:: Python
+   :emphasize-lines: 2-4
+
+    INSTALLED_APPS = [
+        'projects',
+        'neapolitan',
+        'dashboard',
+        [...]
+    ]
+
+Create migrations, and run them::
+
+    python manage.py makemigrations
+    python manage.py migrate
+
+Wire up the admin; in ``projects/admin.py``::
+
+    from .models import Project
+
+    admin.site.register(Project)
+
+and create a superuser::
+
+    python manage.py createsuperuser
+
+Finally, start the runserver and in the admin, add a few ``Project`` objects to the database.
+
+
+Wire up Neapolitan views
+------------------------
+
+Neapolitan expects to extend a base template (its own templates use
+``{% extends "base.html" %}`` so you'll have to provide one at ``dashboard/templates/base.html``::
+
+    {% block content %}{% endblock %}
+
+
+And at the end of ``dashboard/urls.py``::
+
+    from neapolitan.views import CRUDView
+
+    import projects
+
+    class ProjectView(CRUDView):
+        model = projects.models.Project
+        fields = ["name", "owner", "has_tests", "has_docs", "status"]
+
+    urlpatterns += ProjectView.get_urls()
+
+At this point, you can see Neapolitan in action at ``/project/`` (e.g.
+http://127.0.0.1:8000/project/). It won't look very beautiful, but you'll see
+a table of objects and their attributes, along with options to change their values
+(which will work - you can save changes).
+
+
+Next steps
+----------
+
+The default templates use TailwindCSS classes, for styling. You will need to integrate
+TailwindCSS into Django. There is more than one way to do this. The method described here,
+from `Tailwind's own documentation <https://tailwindcss.com/docs/installation/play-cdn>`_,
+is explicitly *not recommended for production*.
+
+Turn your ``base.html`` into a more complete template, and note the ``<script>`` element:
+
+ .. code-block:: html
+   :emphasize-lines: 6
+
+   <!doctype html>
+    <html>
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <script src="https://cdn.tailwindcss.com"></script>
+        </head>
+        <body>
+            {% block content %}{% endblock %}
+        </body>
+    </html>
+
+You notice that the page is now rendered rather more attractively.
+
+Next, Tailwind needs to be able to find all Django's templates so it can target them. We'll
```

Comment:
What is this next work supposed to achieve? I don't see any difference in behaviour or appearance as a result. It's not clear to me what exactly calls that management command, or when.

Reply:
Including the `<script src="https://cdn.tailwindcss.com"></script>` CDN tailwind, makes it just appear nicely straightaway right? (I didn't run it yet.) 

(That's not recommended for production, but it may be OK to defer complexity here... — perhaps with a note saying so. 🤔)